test/algebra-blank-to-var/dawg-syntax/bnodes/syntax-bnodes-02.json

```diff
@@ -4,9 +4,12 @@
     "type": "bgp",
     "patterns": [
       {
+        "type": "pattern",
+        "termType": "Quad",
+        "value": "",
         "subject": {
           "termType": "Variable",
-          "value": "b0"
+          "value": "g_0"
```

Comment:
Why did all these change, is this a SPARQL.js thing?

Reply:
Yeah, I think there was a change in datafactory in SPARQL.js. In any case, the labels shouldn't matter, as long as they are unique within the query (which they still are).